languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,35 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="ACCORD_TOUT_ADJECTIF" name="accord tout adjectif" default="temp_off">
+          <rule>
+              <pattern>
+                  <token postag="(P\+)?D.*" postag_regexp="yes"/>
+                  <token postag="N f s" postag_regexp="yes"/>
+                  <marker>
+                      <token>tout</token>
+                  </marker>
+                  <token postag="J f s" postag_regexp="yes"><exception regexp="yes">^[aeiou].*|&H_ASPIRE;</exception></token>
```

Comment:
The exception is for "a word that doesn't start with a vowel sound", so you're negating all the vowels. Here, you need to include vowels with accents and "h": `<exception regexp="yes">^[aâeéèêioôœuhy].*|&H_ASPIRE;</exception>`. The second element of the exception (after the pipe (|)) will make sure that words starting with h aspiré will be included in the exception nevertheless.

The rule editor did give FPs starting with é... please make these changes and check the rule editor results carefully. Let me know when the rule is ready for merging.